server/storage/cache.go

```diff
@@ -2,14 +2,15 @@ package storage
 
 import (
 	"fmt"
+	"net/url"
 	"strings"
 	"time"
 
 	"github.com/ije/gox/utils"
 )
 
 type Cache interface {
-	Has(key string) (bool, error)
```

Comment:
`Has` better returns `error` since we may need to implement some cache layers by io, for example redis

Reply:
That's a good point, will fix

Reply:
done

Reply:
super！